### ISSUE TITLE
PHP Deprecated warning

### ISSUE THREAD

**Issue by talelmishali** [cknow/laravel-money#76]
Hello,
Just tinkering around with the package, per the documentation, this should (and indeed returning) without a warning:
```php
echo Money::parseByDecimal('1.00', 'USD');
1.00 $
```
But it is returning with a warning:
```php
>>> echo Money::parseByDecimal('1.00', 'USD');
<warning>PHP Deprecated:  Passing a currency as string is deprecated since 3.1 and will be removed in 4.0. Please pass a Money\Currency instance instead. in /Users/main/Code/book/vendor/moneyphp/money/src/Parser/DecimalMoneyParser.php on line 55</warning>
‏1.00 $⏎
>>> 
```

Any reason what am I missing?

**Comment by ricardogobbosouza** [comment]
@talelmishali 

It will be removed in the next version of [money php](https://github.com/moneyphp/money).
https://github.com/moneyphp/money/blob/master/CHANGELOG.md#321---2019-02-07

But I will continue giving support, I will correct it as soon as possible.
Thanks for reporting

**Comment by talelmishali** [comment]
@ricardogobbosouza Thank you!

**Comment by ricardogobbosouza** [comment]
@talelmishali 
Fixed [v6.1.1](https://github.com/cknow/laravel-money/releases/tag/v6.1.1)
